lib/block-editor-settings.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Adds settings to the block editor.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Adds styles and __experimentalFeatures to the block editor settings.
+ *
+ * Note: The settings that are WP version specific should be handled inside the `compat` directory.
+ *
+ * @param array $settings Existing block editor settings.
+ *
+ * @return array New block editor settings.
+ */
+function gutenberg_get_block_editor_settings( $settings ) {
```

Comment:
Documenting what I learn: this is mostly a copy-over of `get_block_editor_settings` but using the `gutenberg_*` functions instead of the `wp_*` ones.